src/components/AddItemButton.tsx

```diff
@@ -0,0 +1,68 @@
+'use client'
```

Comment:
We need to render this component on the client side because it uses `useState` which only works on the browser.